### PR TITLE
Add tooltips for add to group context menu items

### PR DESCRIPTION
**Pull request by zeusongit**
### Purpose

[DYN-4599](https://jira.autodesk.com/browse/DYN-4599)
Adds tooltip to:
- `Add Group to Group` context menu item - `Add selected groups to this group`
- `Add to Group` context menu item - `Add selected nodes to group`

tltp2

### Declarations

Check these if you believe they are true

- [x] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [x] Snapshot of UI changes, if any.

### Reviewers

@DynamoDS/dynamo

### CODE REVIEW

**Review by QilongTang — APPROVED** [review]

**Comment by QilongTang** [comment]
@reddyashish @Amoursol Are you fine with the tooltip change?

**Comment by Amoursol** [comment]
@zeusongit - appears one of the tooltips isn't styled to the new Visual Refresh style (Groups --> Add to Group)

File: src/DynamoCoreWpf/Utilities/NodeContextMenuBuilder.cs [hunk 1]
```diff
@@ -84,6 +84,8 @@ internal static void Build(ContextMenu contextMenu, NodeViewModel nodeViewModel,
         /// <param name="isChecked"></param>
         /// <param name="visibility"></param>
         /// <param name="isEnabled"></param>
+        /// <param name="itemsSource"></param>
+        /// <param name="tooltip"></param>
```
Comment (QilongTang) [inline src/DynamoCoreWpf/Utilities/NodeContextMenuBuilder.cs:88]: these can also be removed right?

File: src/DynamoCoreWpf/Views/Core/AnnotationView.xaml [hunk 2]
```diff
@@ -284,7 +284,8 @@
                           Header="{x:Static p:Resources.ContextUnGroupFromSelection}" />
                 <MenuItem Name="AddGroupToGroup"
                           Command="{Binding AddGroupToGroupCommand}"
-                          Header="{x:Static p:Resources.GroupContextMenuAddGroupToGroup}" />
+                          Header="{x:Static p:Resources.GroupContextMenuAddGroupToGroup}"
+                          ToolTipService.ShowOnDisabled="True" />
```
Comment (QilongTang) [inline src/DynamoCoreWpf/Views/Core/AnnotationView.xaml:288]: I guess we can remove this as well?

**Pull request merged**